app/index.js

```diff
@@ -147,11 +182,130 @@ module.exports = class extends yeoman {
+                    var tempOutFile = "pom.xml";
+                    this.fs.copyTpl(
+                        this.templatePath(this.files[i]),
+                        this.destinationPath(tempOutFile.replace(/src\/main\/java/g, path.join(src, packageFolder))),
+                        { userProps: userProps }
+                    );
+                } else {
+                    this.fs.copyTpl(
+                        this.templatePath(this.files[i]),
+                        this.destinationPath(this.files[i].replace(/src\/main\/java/g, path.join(src, packageFolder))),
+                        { userProps: userProps }
+                    );
+                }
+            }
+        }
+
+        if (showWsdl2Rest) {
+            return wsdl2restGenerate(this.wsdl, this.outdirectory, this.camelDSL, showDebug);
         }
     }
 };
+
+var skipListForWsdl2RestFiles = [
+    'src/main/resources/META-INF/spring/camel-context.xml',
+    'src/main/resources/OSGI-INF/blueprint/blueprint.xml',
+    'pom.xml'
+];
+
+function testFileForWsdl2RestSkip(file) {
+    for(var i=0; i<skipListForWsdl2RestFiles.length; i++) {
+        if (skipListForWsdl2RestFiles[i] == file) return true;
+    }
+}
+
+function wsdl2restGenerate(wsdlUrl, outputDirectory, dsl, isDebug) {
+    if (dsl.indexOf('java') > 0) {
+        console.log(`Generating Rest DSL from SOAP for a Camel Java DSL is currently unsupported.`);
+        return;
+    }
+
+    var wsdl2restdir = path.join(__dirname, 'wsdl2rest');
+    var targetDir = path.join(wsdl2restdir, 'target');
+    var jar = path.join(targetDir, 'wsdl2rest-impl-fatjar-0.1.3-SNAPSHOT.jar');
+
+    var logPath = path.join(wsdl2restdir, 'config', 'logging.properties');
+    var logUrl = fileUrl(logPath);
+
+    var actualJavaOutDirectory = outputDirectory;
+    if (actualJavaOutDirectory.endsWith('/java')) {
+        actualJavaOutDirectory = actualJavaOutDirectory.substring(0, actualJavaOutDirectory.indexOf("/java"));
+    }
+    var outPath = path.join(process.cwd(), actualJavaOutDirectory);
+    var wsdlFileUrl;
+    if (wsdlUrl.startsWith('http')) {
+        wsdlFileUrl = wsdlUrl;
+    } else if (!wsdlUrl.startsWith('file:')) {
+        wsdlFileUrl = fileUrl(wsdlUrl);
+    } else {
+        wsdlFileUrl = wsdlUrl;
+    }
+ 
+    if (!fs.existsSync(outPath)){
+        console.log(`Creating wsdl2rest java output directory`);
+        fs.mkdirSync(outPath);
+    }
+
+    var restContextPath;
+    var rawContextPath;
+    var isBlueprint = dsl.includes('blueprint') > 0;
+    if (isBlueprint) {
+        rawContextPath = "src/main/resources/OSGI-INF/blueprint/blueprint.xml";
+    } else {
+        rawContextPath = "src/main/resources/META-INF/spring/camel-context.xml";
+    };
+    restContextPath = path.join(process.cwd(), rawContextPath);
+
+    // build the java command with classpath, class name, and the passed parameters
+    var cmdString = 'java ';
```

Comment:
what do you think to write it:
```
var cmdString = 'java ' +
                           ' -Dlog4j.configuration=' + logUrl +
                           ' --wsdl ' + wsdlFileUrl +
                           ' --out ' + outPath;
```
see https://google.github.io/styleguide/javascriptguide.xml?showone=Multiline_string_literals#Multiline_string_literals

Reply:
Yes, that's a bit cleaner.